src/basic/verbs.c

```diff
@@ -133,9 +141,28 @@ int dispatch_verb(int argc, char *argv[], const Verb verbs[], void *userdata) {
                         return r;
         }
 
-        if (name)
+        if (create_new_arg) {
+                char **fake;
+                int j;
+
+                fake = new0(char*, left + 1);
+                if (!fake)
+                        return log_oom();
+
+                fake[0] = strdup(verb->verb);
```

Comment:
lacks oom check... that said, why duplicate this at all? you should just be able to cast away the "const" here, and save the copy